src/rgw/rgw_pubsub.cc

```diff
@@ -351,7 +351,10 @@ void rgw_pubsub_bucket_topics::dump(Formatter *f) const
 {
   Formatter::ArraySection s(*f, "topics");
   for (auto& t : topics) {
-    encode_json(t.first.c_str(), t.second, f);
+    auto& topic = t.second.topic;
```

Comment:
this seems related to a different PR?

Reply:
this is to hide the auto-generated topics, but if we decide we don't want to add the original topic to the bucket_notification then we don't need this change

Reply:
wasn't that already done here: https://github.com/ceph/ceph/pull/51138 ?